pkgs/servers/search/manticoresearch/default.nix

```diff
@@ -0,0 +1,79 @@
+{ stdenv, fetchFromGitHub, fetchurl, lib
+, mariadb, mariadb-connector-c
+, boost, cmake, bison, flex
+}:
+
+let
+  columnar = fetchFromGitHub {
+    owner = "manticoresoftware";
+    repo = "columnar";
+    rev = "b211815f";
+    sha256 = "sha256-7oWqa1P1sGUkaTuMLWz36KhR42SjEWQo/4g+JKAsJOU=";
+  };
+  galera_branch = "5e7af6e5";
+  galera = fetchurl {
+    url = "https://github.com/klirichek/galera/archive/${galera_branch}.zip";
+    sha256 = "0520a49m89g2ywphvy2bdd3bwfksa9jyz4gx25aacdnwsp1mxqg9";
+  };
+  re2_branch = "2015-06-01";
+  re2 = fetchurl {
+    url = "https://github.com/manticoresoftware/re2/archive/${re2_branch}.zip";
+    sha256 = "0wr9mpv409568zl97vgxvwimbinqw7j9x6i3jyn7zw6a0lvmfzcb";
+  };
+  stemmer = fetchurl {
+    url = "https://snowballstem.org/dist/libstemmer_c.tgz";
+    sha256 = "1z2xvrjsaaypc04lwz7dg8mjm5cq1gzmn0l544pn6y2ll3r7ckh5";
+  };
+  icu = fetchurl {
+    url = "https://github.com/unicode-org/icu/releases/download/release-65-1/icu4c-65_1-src.tgz";
+    sha256 = "0j6r6qqnhfr5iqkx53k63ifkm93kv1kkb7h2mlgd1mnnndk79qsk";
+  };
+in
+
+stdenv.mkDerivation rec {
+  pname = "manticoresearch";
+  version = "5.0.2";
+
+  src = fetchFromGitHub {
+    owner = "manticoresoftware";
+    repo = "manticoresearch";
+    rev = version;
+    sha256 = "sha256-FRCy/XOlIiScIf5XO/DG5lkBdToT2xrZLsC5TsK5SQw=";
+  };
+
+  nativeBuildInputs = [
+    bison
+    cmake
+    flex
+  ];
+
+  buildInputs = [
+    (boost.override { enableStatic = true; })
+    mariadb.client
+    mariadb-connector-c
```

Comment:
It looks line `mariadb-connector-c` is enough, and `mariadb.client` is needless

Reply:
Thanks, updated